app/models/invite_user_form.rb

```diff
@@ -12,6 +12,7 @@ def initialize(hash)
     @given_name = hash[:given_name]
     @family_name = hash[:family_name]
     @roles = hash[:roles]
+    @team = hash[:team]
```

Comment:
again, no need to be part of the form...

Reply:
Removed